data/json/monsters/turrets.json

```diff
@@ -37,13 +67,13 @@
   {
     "id": "mon_turret_bmg",
     "type": "MONSTER",
-    "name": "antimateriel turret",
-    "description": "Leadworks LLC's T-107A1 Interdictor, an automated .50BMG antimateriel rifle.  Credited with forestalling countless car-bombing attempts, it has always been kept under extremely tight control, so as to itself maintain extremely tight control of its area.  And then the Cataclysm struck.",
+    "name": "M2HB autonomous CROWS II",
```

Comment:
I kinda liked our alternative-history fictional brand names. Maybe leave the `Leadworks LLC's T-107A1 Interdictor` name and just correct the description so it has M2HB instead of a antimateriel rifle?

Reply:
I strongly disagree. Removing most references to leadworks, or at least paring them down so they're a much more *focused* futuretech company  (like silencerco's firearms division, Tracking point, DSR-precision, Cheytac International or KRISS-USA).  New firearms, in real life, don't often work 100%, so its very jarring to see such a huge variety of stuff that works 110%.

Reply:
I just figured something was up in Leadworks R&D department, like maybe they've got an AI, sort of a private-sector prototype of Melchior, doing design work or something. Do we know who the contractor was who got hired to build Melchior?